source/api_documentation/network_integration/customer_phone_numbers/index.rst

```diff
@@ -114,3 +114,15 @@ Endpoint:
    :path: /customer_phone_numbers/&lt;customer_phone_number_id&gt;
    :description: Delete a Destination
    :page: delete_customer_phone_number
+
+.. raw:: html
+
+  <br/>
+  <br/>
+  <div class="alert alert-info">
+    <b>NOTE: Creating & Deleting Destinations</b></br>
+    Creating a Destination via this endpoint isn't possible.  One can be made alongside a RingPool that it'll be associated with via the RingPool endpoint (<a href="https://developers.invoca.net/en/2022-08-01/api_documentation/network_integration/ringpools/index.html"><b>Documentation</b></a>).<br/>
```

Comment:
🔧 `One can be made alongside a RingPool that it'll be associated with via the RingPool endpoint` -> `A Destination can be made alongside a RingPool that it'll be associated with via the RingPool endpoint`